apps/studio/src/features/editing-experience/components/TipTapComponent.tsx

```diff
@@ -100,6 +112,13 @@ function TipTapComponent({ type, data, path }: TipTapComponentProps) {
     content: data,
     onUpdate({ editor }) {
       // TODO: set editor state - content is retrieved via editor.getJSON().content
+      const { content } = editor.getJSON()
+
+      if (validateFn(content)) {
+        const newPageState = [...savedPageState]
+        newPageState[blockIndex] = content
+        setPreviewPageState(newPageState)
+      }
```

Comment:
we don't need this validation here because internally, tiptap's schema conforms to our schema already. 

we also shouldn't do this because we are validating the full content recursively

Reply:
Hmm I think we are not entirely guaranteed this fact cos we manually ensure that our schema conforms to Tiptap's schema, but yeah removing it for now cos we are already validating the full content.